src/backend/dx11/src/lib.rs

```diff
@@ -3150,6 +3174,17 @@ pub struct InternalImage {
     debug_name: Option<String>
 }
 
+impl InternalImage {
+    unsafe fn release_resources(&mut self) {
+        (&*self.raw).Release();
+        self.copy_srv = None;
+        self.srv = None;
+        self.unordered_access_views.clear();
```

Comment:
we aren't calling `Release` on each view? This seems different from the `InternalBuffer` logic

Reply:
They're owned; they're `Option<ComPtr<Whatever>>`